.github/workflows/ci-earthly-next-docker-ubuntu.yml

```diff
@@ -0,0 +1,209 @@
+name: Earthly Next Docker CI Ubuntu
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - 'docs/**'
+      - '**.md'
+      - '.github/renovate.json5'
+      - '.github/CODEOWNERS'
+      - 'LICENSE'
+  pull_request:
+    branches: [ main ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  # this job will output a boolean value to check whether files that require these tests to run
+  # since all jobs depend on `build-earthly` job, conditionally running it will either cause all jobs to run or skip,
+  # depending on the output value of this job
+#  check-essential-files:
```

Comment:
Do we need to retain this?

Reply:
good point; it was copy/pasted but I don't think it applies to us. I'll remove it.